src/Command/ModifyEntity/LineItem/LineItemChangeStateCommand.php

```diff
@@ -171,11 +181,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
                 $this->logger->info(
                     sprintf(
-                        'The operation: "%s" was executed for Line Item with id: "%d"',
+                        "The operation: '%s' was executed for Line Item with id: '%s'",
                         $toggle,
-                        $lineItem->getId()
-                    ),
-                    ['slug' => $lineItem->getSlug(), 'uri' => $lineItem->getUri()]
+                        (string)$lineItem->getId()
+                    )
```

Comment:
Why did you remove the context?

Reply:
I dunno Dani's opinion on this, but as for me, _in this_ command we already have more than enough context. How do you think?

Reply:
Yes, that was the reason, line item id is enough in the message, the rest of the context was not relevant here (because we change only the status)